free-1.md

```diff
@@ -10,41 +10,139 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+Classical and baroque music often feature musical 
+[ornaments](https://en.wikipedia.org/wiki/Ornament_(music)), such as trills, 
+mordants, and turns. These serve as embellishments to the overall melody.
+However, ornaments are quite ambiguous in how people should play them. 
+Different composers interpret them differently, and even time period affects 
+when ornaments should be played in relation to the beat. Unless musicians are 
+well-versed in music history, they may not realize the specific interpretations 
+desired by each composer. 
+
+Of course, people might argue that music should be left ambiguous because music 
+is a subjective art. However, if musicians want to play according to a 
+composer's style, they must learn and understand the different interpretations 
+of musical ornaments.
+
+I want to help musicians hear the subtle differences among various 
```

Comment:
This could also be problematic, if we're looking at Baroque music, for example, than it was probably designed for a harpischord (sp?) rather than a piano, and although we can mimic the sound using a piano, it won't quite be exact. Would you handle different instruments as well?